src/main/java/info/tehnut/soulshards/core/config/ConfigSoulShards.java

```diff
@@ -15,14 +14,14 @@
 
     private static MultiblockPattern multiblock;
 
-    private ConfigBalance balance;
-    private ConfigClient client;
-    private ConfigEntityList entityList;
+    private static ConfigBalance balance;
+    private static ConfigClient client;
+    private static ConfigEntityList entityList;
```

Comment:
Why were these made static..? This won't work with Gson

Reply:
I already asked about this on discord

Reply:
I can't figure this out. ConfigSoulShards has the ConfigEntityList variable-- when the game starts, it's initialized with a size of 64. As far as I can tell it's never accessed or touched again. When the game starts and the same instance of the class is accessed, the list has a size of 0. Going to keep messing with it for now.